S2VX.Game/Editor/NotesTimeline.cs

```diff
@@ -214,20 +242,29 @@ public class NotesTimeline : CompositeDrawable {
             }
         }
 
-        protected override void Update() {
-            TickBar.Clear();
-            TickBar.Add(new RelativeBox {
-                Colour = Color4.White,
-                Height = TimelineHeight / 3.5f,
-            });
+        private void AddVisibleNotes() {
+            var lowerBound = Story.GameTime - SectionLength * 1000 / 2;
```

Comment:
Were you going to have named consts for some of these magic numbers?

Reply:
Yea, let me do that for 1000 Rn